components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/utils/Constants.java

```diff
@@ -88,6 +87,17 @@
     public static final String NEVER_REACHED = "Never Reached";
     public static final int DEFAULT_TIME_INTERVAL_MILLIS = 300000;
 
+    public static final String MANAGERID = "MANAGERID";
+    public static final String WORKERID = "WORKERID";
+    public static final String HOST = "HOST";
+    public static final String PORT = "PORT";
+    public static final String INTEGER_TYPE = "integerType";
```

Comment:
what integer type?? be more descriptive. instead of suffixing, prefix them like `TYPE_INTEGER`